swarm/runtime/stepwise/routing/__init__.py

```
@@ -116,3 +123,24 @@
     # Candidate-set pattern
     "generate_routing_candidates",
 ]
+
+
+def __getattr__(name: str):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `__getattr__`

(ANN202)